javascript/javascript1/week3/note_taking.js

```diff
@@ -0,0 +1,76 @@
+// NOnoN0nOYes (Note taking app)
+const notes = [];
+
+// Save a note
+function saveNote(content, id) {
+    let noteObject = {};
+    noteObject.content = content;
+    noteObject.id = id;
+    notes.push(noteObject);
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+console.log(notes);
+
+// Get a note
+function getNote(id) {
+    let noteIndex;
+    let noteContent = "";
```

Comment:
Do you need this variable? what if you return value instead of assigned to noteContent? like:

Reply:
Yes, indeed